Finance/teller-track-spending.1h.sh

```diff
@@ -7,7 +7,7 @@
 # - teller-cli: https://github.com/sebinsua/teller-cli#from-release
 # - pcregrep: `brew install pcre`
 #
-# <bitbar.title>teller-track-spending</bitbar.title>
+# <bitbar.title>Track spending at UK banks</bitbar.title>
```

Comment:
We already have a precedent of some fairly long descriptions, see [the pull-requests script](https://github.com/matryer/bitbar-plugins/blob/master/Dev/GitHub/pull-requests.5m.sh#L7) :smile: .